pkgs/development/libraries/liblinear/default.nix

```diff
@@ -1,35 +1,37 @@
-{ stdenv, fetchurl, fixDarwinDylibNames }:
+{ stdenv, fetchFromGitHub, fixDarwinDylibNames }:
 
-stdenv.mkDerivation rec {
+let
+  soVersion = "4";
```

Comment:
Do you know what's upstream policy regarding this? Why wouldn't they use e.g `liblinear.${version}.so`?

Reply:
I think they just bump the `SONAME` when there is an ABI change, but they don't bump the major version.